README.md

```diff
@@ -5,6 +5,8 @@ Specification to allow holders to request issuance of credentials and issuers to
 
 The current WG-Draft version is built automatically from the master branch and can be accessed at: https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html
 
+Other versions of the spec can be accessed at: https://openid.net/sg/openid4vc/specifications/
```

Comment:
Do we want to clarify this is a WG's draft vs editors' draft?

Reply:
Not entirely sure what you're suggesting we change? line 6 does say that it's a working group draft.